File: qt-app/py/src/mainwindow.py
from ui_mainwindow import Ui_MainWindow
from PySide2.QtWidgets import *


class MainWindow(QMainWindow, Ui_MainWindow):

    def __init__(self):
        super(MainWindow, self).__init__()

        self.setupUi(self)
        # Connect button signal to appropriate slot
        self.myButton.released.connect(self.myButtonPushed)

    def myButtonPushed(self):
        self.myTextEdit.append("Lorum ipsum")

File: qt-app/cpp/src/mainwindow.cpp
//
// Created by Gilbert François Duivesteijn on 20.11.18.
//

#include "mainwindow.h"
#include "../ui/ui_mainwindow.h"

#include <QtWidgets>

MainWindow::MainWindow(QWidget *parent)
        : QMainWindow(parent), ui(new Ui::MainWindow) {

    ui->setupUi(this);
    // Connect button signal to appropriate slot
    connect(ui->myButton, SIGNAL (released()), this, SLOT (myButtonPushed()));
}

void MainWindow::myButtonPushed() {
    ui->myTextEdit->append("Lorum ipsum");
}

MainWindow::~MainWindow() {
    delete ui;
}

